File: lib/layer2_core/dark/e_error_container_dark.dart
import '../../layer1_raw/design_token.dart';
import '../../layer1_raw/dark/system.dart';
import '../../layer1_raw/dark/standard.dart';
import '../../layer1_raw/dark/blue.dart';
import '../../layer1_raw/dark/green.dart';
import '../../layer1_raw/dark/red.dart';
import '../../layer1_raw/dark/nothing.dart';

class EErrorContainerDark {
  static DesignToken getToken(dynamic activeTheme) {
    switch (activeTheme.toString().split('.').last) {
      case 'system': return csyd43;
      case 'standard': return cstd43;
      case 'blue': return cbd43;
      case 'green': return cgd43;
      case 'red': return crd43;
      case 'nothing': return cnod43;
      default: return cstd43;
    }
  }
}

File: android/src/main/kotlin/com/example/design_engine/layer2_core/dark/EErrorContainerDark.kt
package com.example.design_engine.layer2_core.dark
import com.example.design_engine.R
import com.example.design_engine.layer3_logic.AppTheme
object EErrorContainerDark {
    fun getResId(activeTheme: AppTheme): Int {
        return when (activeTheme) {
            AppTheme.SYSTEM -> R.string.csyd43
            AppTheme.STANDARD -> R.color.cstd43
            AppTheme.BLUE -> R.color.cbd43
            AppTheme.GREEN -> R.color.cgd43
            AppTheme.RED -> R.color.crd43
            AppTheme.NOTHING -> R.color.cnod43
        }
    }
}
